### DISCUSSION TITLE
请问一下 审核过程中是可以刷新页面 但是执行后 页面不刷新 这个有什么办法吗

### DISCUSSION THREAD

**Discussion by LARcenCielGIT** [hhyo/Archery#2228] [Q&A]
如题，有配置 或者修改一下这个页面

**Comment by LeoQuote** [comment]
能详细谈一下你的需求吗? 执行之后为什么还要刷新工单状态?

**Reply by LARcenCielGIT** [reply]
执行之后页面还是显示排队中，但是sql已经执行成功，然后手动刷新页面才会显示执行成功，这里有办法做到获取状态自动刷新吗

**Reply by LARcenCielGIT** [reply]
大佬， 我补充一下 我在机器A部署了完成之后  把项目的代码部署到B了 不过数据资源还在A上，这时候在B执行sql的时候就会出现上述问题，在机器A上是正常的，这个是什么原因没弄明白

**Reply by LeoQuote** [reply]
这不是我们推荐的部署方式哈, 出了问题你还是得自己在研究研究

**Reply by LARcenCielGIT** [reply]
哦哦哦 好的
